src/Core/src/Handlers/ImageButton/ImageButtonHandler.iOS.cs

```diff
@@ -60,6 +60,9 @@ public override void SetImageSource(UIImage? platformImage)
 				if (Handler?.PlatformView is not UIButton button)
 					return;
 
+				if (platformImage?.Images is not null && platformImage.Images.Length > 0)
+					platformImage = platformImage.Images[0];
```

Comment:
Not currently. The button does not have the Images property, but in a separate PR I will try find the underlying UIImageView and set it up correctly.